mindmeld/text_preparation/normalizers.py

```diff
@@ -127,7 +128,7 @@ class NFD(Normalizer):
     """Unicode NFD Normalizer Class."""
```

Comment:
Should we add examples in the docstrings of the normalizer classes to differentiate between them?

Reply:
The examples are better described in images and are a bit too complex to be explained in doc-strings. We can instead direct them to Unicode's official documentation on the Normalization type. Updated the class docstrings to the following:

```
class NFKC(Normalizer):
    """Unicode NFKC Normalizer Class.
    (Compatibility Decomposition, followed by Canonical Composition)

    For more details: https://unicode.org/reports/tr15/#Norm_Forms
    """ 
```